site/src/Librecores/ProjectRepoBundle/Validator/Constraints/UserOrgNameValidator.php

```diff
@@ -76,7 +76,7 @@ class UserOrgNameValidator extends ConstraintValidator
      *
      * @var bool
      */
-    const CHECK_UNIQUE_ORG  = true;
+    const CHECK_UNIQUE_ORG  = false;
```

Comment:
Sounds good. Just on a timing note, I'd love to get this integrated before FOSDEM next weekend, so maybe you can just update this PR with what you have tonight and then I can take it from there if I find more free time than you do. Possibly then Bernd can also have a look at the styling before next weekend, which would be fantastic.